pyaedt/modules/Boundary.py

```diff
@@ -317,10 +317,21 @@ def __init__(self, app, name, props, boundarytype):
         self._app = app
         self._name = name
         self.props = BoundaryProps(self, OrderedDict(props))
-        self.type = boundarytype
+        self._type = boundarytype
         self._boundary_name = self.name
         self.auto_update = True
 
+    @property
+    def type(self):
```

Comment:
The property is missing a docstring which is causing the documentation build to fail.
As soon as it is added, the workflow should pass entirely.
```suggestion
    def type(self):
        """Boundary type."""
```